torchvision/transforms/v2/_misc.py

```diff
@@ -234,13 +235,15 @@ class ToDtype(Transform):
             A dict can be passed to specify per-datapoint conversions, e.g.
             ``dtype={datapoints.Image: torch.float32, datapoints.Video:
             torch.float64}``.
+        scale:
     """
 
     _transformed_types = (torch.Tensor,)
 
-    def __init__(self, dtype: Union[torch.dtype, Dict[Type, Optional[torch.dtype]]]) -> None:
+    def __init__(self, dtype: Union[torch.dtype, Dict[Type, Optional[torch.dtype]]], scale: bool = False) -> None:
         super().__init__()
         if not isinstance(dtype, dict):
+            # TODO: only care about images or videos when passing a single dtype?
```

Comment:
As discussed offline @pmeier , we need to decide on this

Reply:
IMO, that is fine, since that is what most people likely want to do. However, should we also adapt `scale: Optional[bool] = None` in that case to mean: "if you only pass a single dtype, we are going to assume you want to convert and scale images and videos"? Basically having 

```python
if scale is None:
    scale = isinstance(dtype, torch.dtype)
self.scale = scale
```

in the constructor. That way `ToDtype` becomes a drop-in replacement for `ConvertImageDtype` with the ability to finetune.

Reply:
> should we also adapt scale

This is the one thing where I think we should diverge from `ConvertImageDtype`: one of the most confusing thing about it is that it scales, while there's absolutely nothing in its name that makes it obvious. From https://github.com/pytorch/vision/issues/7756 

> It should be absolutely clear that ConvertDtype also scales the values

I'd rather enforce that users must pass `scale=True` if they want scaling to happen. Any concern?

Reply:
I implemented what we agree on in [00aef9b](https://github.com/pytorch/vision/pull/7759/commits/00aef9b038f723df474a56152094543231fe615c) (leaving-out the `scale` thing for now)